### ISSUE TITLE
Vendor package name change?

### ISSUE THREAD

**Issue by camfindlay** [silverstripe/fluent-regional#12]
It's currently silverstripe/regional should probably be more verbose  like silverstripe/fluent-regional

Thoughts @stevie-mayhew ? Would be good to address before it gets packagist registered.

**Comment by stevie-mayhew** [comment]
@camfindlay yeah I agree that makes much more sense :)

**Comment by camfindlay** [comment]
Any good way to address the name change if someone has already used (plus repo config) in their composer.json?

**Comment by stevie-mayhew** [comment]
I can't think of anything apart from putting in a `replaces` section in composer.json

Peoples stuff will still break through :cry:

**Comment by robbieaverill** [comment]
@camfindlay @stevie-mayhew yeah `replace` is correct. If you bundle it in with an SS4 upgrade then it'll be less dangerous.
